main/manager.c

```diff
@@ -6248,6 +6254,10 @@ static int action_originate(struct mansession *s, const struct message *m)
 	pthread_t th;
 	int bridge_early = 0;
 
+    if(!ast_strlen_zero(gosub)){
+        predial_callee = gosub ;
```

Comment:
initially I tried to always initialize the variable that would be passed to the function to make the call, but the function itself that parses the parameter already guarantees that it will not be Null. I made the adjustment and new commit.
Thanks for indication